doc/rtd/reference/availability.rst

```diff
@@ -17,19 +17,39 @@ Distributions
 ``Cloud-init`` has support across all major Linux distributions, FreeBSD,
 NetBSD, OpenBSD and DragonFlyBSD:
 
+- AlmaLinux
 - Alpine Linux
+- AOSC OS
```

Comment:
Please add Amazon Linux 2023 here too as there is [cloud-init support for AL2023](https://docs.aws.amazon.com/linux/al2023/ug/cloud-init.html)
```suggestion
- AOSC OS
- Amazon Linux 2023
```